bin/main.rs

```diff
@@ -75,6 +102,48 @@ struct Args {
         env = "WADM_MAX_JOBS"
     )]
     max_jobs: usize,
+
+    /// URL of the Nats Jetstream KV that will be used for storage
+    #[arg(long = "storage-nkv-url", env = "STORAGE_NATSKV_URL")]
+    storage_nkv_url: Option<String>,
+
+    /// Optional prefix for the server tate that will be stored.
+    /// this prefix will be prepended to the bucket name of the lattice
+    /// (ex. <prefix>_lattice_<lattice id>).
+    #[arg(
+        long = "storage-nkv-lattice-bucket-prefix",
+        env = "STORAGE_NATSKV_LATTICE_BUCKET_PREFIX"
+    )]
+    storage_nkv_lattice_bucket_prefix: Option<String>,
+
+    /// (Optional) NATS NKey authentication
+    #[arg(
+        long = "storage-nkv-nats-auth-nkey",
+        env = "STORAGE_NATSKV_NATS_AUTH_NKEY"
+    )]
+    storage_nkv_nats_auth_nkey: Option<String>,
+
+    /// (Optional) NATS credential file to use when authenticating
+    #[arg(
+        long = "storage-nkv-nats-auth-creds-file",
+        env = "STORAGE_NATSKV_NATS_AUTH_CREDS_FILE"
+    )]
+    storage_nkv_nats_auth_creds_file: Option<String>,
+
+    /// (Optional) NATS JWT seed to use when authenticating
+    #[arg(
+        long = "storage-nkv-nats-auth-jwt-seed",
+        env = "STORAGE_NATSKV_NATS_AUTH_JWT_SEED"
+    )]
+    storage_nkv_nats_auth_jwt_seed: Option<String>,
+
+    /// (Optional) NATS JWT file to use when authenticating
+    #[arg(
+        long = "storage-nkv-nats-auth-jwt-path",
+        env = "STORAGE_NATSKV_NATS_AUTH_JWT_PATH"
+    )]
+    storage_nkv_nats_auth_jwt_path: Option<String>,
+>>>>>>> 0907ff2 (feat: track lattice state in KV)
```

Comment:
Looks like there's a bit of a merge conflict still sitting in this file?

Reply:
Ah sorry, just fixed this!